browser/js/categories/category.controller.js

```diff
@@ -1,4 +1,26 @@
-app.controller('CategoryCtrl',function($scope, allCategoryKits, title){	
+app.controller('CategoryCtrl',function($scope, allCategoryKits, title, AuthService, $uibModal){
```

Comment:
The CategoryCtrl is very similar to the KitsController.  Could you perhaps find a way to have the KitsController apply to both All Kits and single Category pages?  To help keep our code dry